src/ProjectNeon/Assets/Features/Effects/Attack.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public sealed class Attack 
+{
+    private Member attacker;
+    public Member Attacker { get { return this.attacker; } }
```

Comment:
No need for a field. Just make a readonly property and assign it in the `ctor`

`public Member Attacker { get; }` is allowed to be set once in the `ctor`

Reply:
@SilasReinagel Right, corrected, thanks for the tip